pkg/proxy/userspace/proxier_test.go

```diff
@@ -429,6 +429,9 @@ func TestTCPProxyStop(t *testing.T) {
 	if err != nil {
 		t.Fatalf("error adding new service: %#v", err)
 	}
+	if atomic.LoadInt32(&svcInfo.isAlive) != 1 {
```

Comment:
Also, it would be good to capture this in a variable, to ensure that your log message sees the same thing the if statement sees.